io.github.babaissarkar.ssplot.json

```
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.github.babaissarkar.ssplot",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "ssplot-run",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=xdg-documents:rw"
+    ],
+    "modules": [
+        {
+            "name": "SSPlot",
+            "buildsystem": "simple",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/babaissarkar/ssplot/archive/refs/tags/v2.2.2.tar.gz",
+                    "sha256": "3582d04d8d8be2b71ef5104a0d888f505e54b7df3c155647babd3ab6aabde247",
+                    "dest-filename": "ssplot.tar.gz"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/babaissarkar/ssplot/v2.2.2/SSPlot.AppDir/ssplot.desktop",
+                    "sha256": "bf300c2d694417732053f5930ef8eb3063630c4abcd8afb3d5a5129f07c66901",
+                    "dest-filename": "io.github.babaissarkar.ssplot.desktop"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/babaissarkar/ssplot/v2.2.2/io.github.babaissarkar.ssplot.metainfo.xml",
+                    "sha256": "8eb33e09286f2e271b8b2ba98f0ea1f32d0ebcc81b24ebd2e1b09b25e7054300"
+                },
+                {
+                    "type": "file",
+                    "path": "ssplot-run"
+                }
+            ],
+            
+            "build-commands": [
+                 "sed -i 's/Icon=ssplot/Icon=io.github.babaissarkar.ssplot/' io.github.babaissarkar.ssplot.desktop",
```

never use `sed`, it has no failure mode
```suggestion
                 "desktop-file-edit --set-icon=$FLATPAK_ID SSPlot.AppDir/ssplot.desktop",
```